layouts/partials/footer.html

```diff
@@ -15,7 +15,9 @@
       {{ end }}
       {{ if or (.Site.Params.credit) (eq (isset .Site.Params "credit") false) -}}
       <div class="col-xs">
-          <small>Theme: <a href="https://github.com/kdevo/osprey-delight" target="_blank" rel="noopener">Osprey Delight</a></small>
+          <!--
+          <small>Theme: <a href="https://github.com/jeffhx/osprey-delight" target="_blank" rel="noopener">Osprey Delight</a></small>
```

Comment:
For me it's a bit discouraging to not receive any credit at all.
Neverheless, I've left it completely configurable: You can set the `.Params.credit` in your site's config to `false` and the attribution will disappear (take a look at the condition a few lines above).

I think I've kept the attribution very short and unobtrusive already, so I'm wondering: What's the reasoning behind removing it completely?

Reply:
Hello Kai, thanks for the reply. I was not aware of this configure, thanks for pointing it out. 
The reasoning removing it completely is i plan to use it in a production site.

Sorry this discourages you in any way. It was not my intent.

Reply:
No problem, I understand that it was not your intent. Thanks for your answer!